src/calculateRentalCost.js

```diff
@@ -3,8 +3,21 @@
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
   // write code here
+  const dayCost = 40;
+  const discount3 = 20;
+  const discount7 = 50;
+  const totalAmount = days * dayCost;
+
+  if (days < 3) {
```

Comment:
Use constants to define the minimum days for discounts to avoid magic numbers. For example, 'const SHORT_TERM_RENTAL = 3;' and 'const LONG_TERM_RENTAL = 7;'.